XPlat/AzureMessaging/iOS/source/ApiDefinition.cs

```diff
@@ -156,5 +156,99 @@ public partial interface SBTokenProvider
 		[Export ("setTokenWithRequest:error:")]
 		bool SetToken (NSMutableUrlRequest request, out NSError error);
 	}
+
+	[BaseType (typeof (NSObject))]
+ 	public partial interface MSNotificationHub 
+ 	{
+ 		[Static, Export ("initWithConnectionString:hubName:")]
+ 		void Init (string connectionString, string hubName);
+
+ 		[Static, Export ("setDelegate:")]
+ 		void SetDelegate ([NullAllowed] MSNotificationHubDelegate nubDelegate);
+
+ 		[Static, Export ("setTemplate:forKey:")]
+ 		void SetTemplate (MSInstallationTemplate template, string key);
+
+ 		[Static, Export ("removeTemplate:")]
+ 		void RemoveTemplate (string key);
+
+ 		[Static, Export ("getTemplate:")]
+ 		MSInstallationTemplate GetTemplate (string key);
+
+ 		[Static, Export ("addTag:")]
+ 		bool AddTag (string tag);
+
+ 		[Static, Export ("addTags:")]
+ 		bool AddTags (NSArray<NSString> tags);
+
+ 		[Static, Export ("removeTag:")]
+ 		bool RemoveTag (string tag);
+
+ 		[Static, Export ("removeTags:")]
+ 		bool RemoveTags (NSArray<NSString> tags);
+
+ 		[Static, Export ("getTags")]
+ 		NSArray<NSString> GetTags ();
+
+ 		[Static, Export ("clearTags")]
+ 		void ClearTags ();
+
+ 		[Static, Export ("getInstallation")]
+ 		MSInstallation GetInstallation ();
+
+ 		[Static, Export ("didRegisterForRemoteNotificationsWithDeviceToken:")]
+ 		void DidRegisterForRemoteNotifications (NSData deviceToken);
+
+ 		[Static, Export ("didFailToRegisterForRemoteNotificationsWithError:")]
+ 		void DidFailToRegisterForRemoteNotifications (NSError error);
+
+ 		[Static, Export ("didReceiveRemoteNotification:")]
+ 		bool DidReceiveRemoteNotification (NSDictionary userInfo);
+ 	}
+
+ 	[BaseType (typeof (NSObject))]
+ 	public partial interface MSInstallation 
+ 	{
+ 		[Export ("installationID")]
+ 		string InstallationID { get;set; }
+
+ 		[Export ("pushChannel")]
+ 		string PushChannel { get;set; }
+
+ 		[Export ("platform")]
+ 		string Platform { get;set; }
+
+ 		[Export ("tags")]
+ 		NSArray<NSString> Tags { get;set; }
+ 	}
+
+ 	[BaseType (typeof (NSObject))]
+ 	public partial interface MSInstallationTemplate 
```

Comment:
Let's remove the MSInstallationTemplate for now since it's not being used and is currently empty